vhdbuilder/packer/configure-windows-vhd.ps1

```diff
@@ -206,14 +206,20 @@ function Get-FilesToCacheOnVHD {
             "https://acs-mirror.azureedge.net/kubernetes/v1.18.10-hotfix.20210118/windowszip/v1.18.10-hotfix.20210118-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.18.14-hotfix.20210322/windowszip/v1.18.14-hotfix.20210322-1int.zip",
             "https://acs-mirror.azureedge.net/kubernetes/v1.18.17-hotfix.20210322/windowszip/v1.18.17-hotfix.20210322-1int.zip",
+            "https://acs-mirror.azureedge.net/kubernetes/v1.18.17-hotfix.20210505/windowszip/v1.18.17-hotfix.20210505-1int.zip",
```

Comment:
We have two 1.18.17 package

Reply:
with different hotfixes

Reply:
Shouldn't we use one hotfix for each version? How will use different hotfixes of one k8s version in RP?

Reply:
I think that it is ok to cache 2 different hotfixes. We can remove it next time.
1. We may not publish VHD and rp k8s version changes at the same time
2. All cached files will be deleted after provisioning Windows VMs

Reply:
I agree with item 2.
But for item 1,  just take the next latest rp release for example, aren't we going to use  the old one `v1.18.17-hotfix.20210322` instead of the new one `v1.18.17-hotfix.20210505`? I think it's in the plan to use the new one.

Reply:
Just in case. Sometimes we may need to hotfix RP release with the new VHD but the commit in RP to change the k8s version may not be included in the hotfix so this is more flexible.

Reply:
We definitely only need one package for one version if this is in one commit with RP change.

Reply:
Could you please add a comment in the PR for this idea? Because people will follow one k8s hotfix package in one VHD if they take previous PRs to onboard k8s hotfix for reference.

Reply:
The case happens when we cache k8s hotfixes plus other import resources and as k8s hotfix relies on other changes like Linux side onboarding a new k8s version, and is blocked by that work. we can continue to ship our other import resources without k8s hotfixes.